src/Controllers/LoginController.php

```diff
@@ -58,7 +55,7 @@ public function loginAction()
     /**
      * Logs the current user out.
      *
-     * @return Response|string
+     * @return RedirectResponse
      */
     public function logoutAction()
```

Comment:
```suggestion
    public function logoutAction(): RedirectResponse
```
And we can remove `@return` above.